src/libraries/System.ServiceModel.Syndication/src/System/ServiceModel/Syndication/Atom10FeedFormatter.cs

```diff
@@ -137,11 +137,11 @@ internal static void ReadCategory(XmlReader reader, SyndicationCategory category
                     {
                         category.Name = reader.Value;
                     }
-                    else if (reader.LocalName == Atom10Constants.SchemeTag && reader.NamespaceURI == string.Empty)
+                    else if (reader.LocalName == Atom10Constants.SchemeTag && reader.NamespaceURI.Length == 0)
```

Comment:
Is XmlReader.NamespaceUri guaranteed to return non-null?

Reply:
```suggestion
                    else if (reader.LocalName == Atom10Constants.SchemeTag && reader.NamespaceURI == string.Empty)
```